manipulate_pim_data/rule/general_information_on_rule_format.rst

```diff
@@ -551,6 +551,96 @@ The expected values are:
             - type: clear
               field: associations
 
+Calculate
+_________
+
+This action allows to calculate numeric attribute values, with simple mathematical operations (addition, substraction, multiplication and division)
+
+This action only accepts number, measurement or price collection attributes for both source and destination.
```

Comment:
```suggestion
This action requires number, measurement or price collection attributes for both sources and destinations.
```

Reply:
but there is only 1 source and 1 destination. Maybe rather:
`This action requires number, measurement or price collection attributes for both the source and the destination.` ?

Reply:
ok